hack/install.sh

```diff
@@ -46,13 +46,13 @@ has_pacman() {
 install_required_packages() {
   if $(has_apt_get); then
     $SUDO apt-get update -y
-    $SUDO apt-get install -y curl runc bridge-utils
+    $SUDO apt-get install -y curl runc bridge-utils iptables
   elif $(has_yum); then
     $SUDO yum check-update -y
-    $SUDO yum install -y curl runc
+    $SUDO yum install -y curl runc iptables
   elif $(has_pacman); then
     $SUDO pacman -Syy
-    $SUDO pacman -Sy curl runc bridge-utils
+    $SUDO pacman -Sy curl runc bridge-utils iptables
```

Comment:
Same comment as per above.

In the worst case, this PR breaks both Red Hat / CentOS and Arch Linux - package names can very, especially in Arch Linux.

Maintainers cannot trust untested changes, even those made with good intentions.

Please revert both.